lib/fluent/plugin/in_forward.rb

```diff
@@ -125,7 +125,12 @@ def on_message(msg, chunk_size, source)
         return
       end
 
-      # TODO format error
+      # TODO: raise an exception if broken chunk is generated by recoverable situation
+      unless msg.is_a?(Array)
+        log.warn "incoming chunk is broken:", source: source, msg: msg
```

Comment:
`inspect` is done in log methods.

https://github.com/fluent/fluentd/blob/7bd529cc8007f04592c971f6b2fd9d216c60cf5d/lib/fluent/log.rb#L279